modules/core/src/webgl/framebuffer.js

```diff
@@ -668,8 +669,17 @@ export default class Framebuffer extends Resource {
   }
 
   _unattach({attachment}) {
```

Comment:
Nit/Optimization opportunity: Since private method with few params, no need to incur overhead of named parameters.